sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -188,7 +188,8 @@ def __init__(self,
             self.name_context.step_name,
             0,
             next(iter(itervalues(consumers))),
-            self.windowed_coder)
+            self.windowed_coder,
+            self)
```

Comment:
Okay, I refactored it to pass only the necessary data. Also, I thought self was just a reference to the class instance so it wouldn't be expensive to pass it around - is this not the case?

Reply:
When I said "big" I meant in terms of surface area. (You're right that there's no performance concern here.) E.g. it's easier to reason about the code when we see that the callee is interested in the type hints rather than the whole operation.